README.md

```diff
@@ -25,6 +25,18 @@ The Terra Azure Relay Listener establishes a bi-directional channel with Azure R
 
 `listener.corsSupportProperties.preflightMethods` Methods that we support CORS. Default to `OPTIONS, POST, PUT, GET, DELETE, HEAD, PATCH`.
 
+### Sam Inspector config options
+
+By using the Sam Checker inspector, the Listener can be configured to allow access only for users
+that have write access to a specific Sam resource.
+
+`listener.samInspectorProperties.samUrl`: URL to the Sam instance we should talk to
+
+`listener.samInspectorProperties.samResourceId`: The id of the Sam resource to check access to
+
+`listener.samInspectorProperties.samResourceType`: The type of the Sam resource to check access to.
+Defaults to `controlled-application-private-workspace-resource`, which corresponds to Jupyter Notebooks.
```

Comment:
I think this corresponds to leo managed resources...for cromwell, it'll be the same when managed through leonardo